api/diagnostics.go

```diff
@@ -771,7 +773,9 @@ type CommandProvider struct {
 }
 
 func loadExternalProviders(cfg *config.Config) (externalProviders LogProviders, err error) {
+	logrus.Infof("loadExternalProviders:")
 	// return if cfg file not found.
+	logrus.Infof("cfg.FlagDiagnosticsBundleEndpointsConfigFile: %s", cfg.FlagDiagnosticsBundleEndpointsConfigFile)
```

Comment:
please remove the Info message or turn them into debug

Reply:
I will replace it with Debugf